restart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ENGLISH">
+<head>
+    <meta charset="UTF-8">
+    <title> Becks Webpage</title>
+    <link rel="stylesheet" href="restart.css" type="text/css"/>
+</head>
+<style> 
```

Comment:
Separation of concerns. When we separate our HTML and CSS into separate files, it's usually better to stick to that separation throughout the code, rather than having some inline CSS and some imported CSS.